common/scala/src/main/scala/org/apache/openwhisk/core/database/ActivationFileStorage.scala

```diff
@@ -111,8 +111,16 @@ class ActivationFileStorage(logFilePrefix: String,
   def activationToFile(activation: WhiskActivation,
                        context: UserContext,
                        additionalFields: Map[String, JsValue] = Map.empty) = {
-    val transcribedLogs = transcribeLogs(activation, additionalFields)
-    val transcribedActivation = transcribeActivation(activation, additionalFields)
+    activationToFileExtended(activation, context, additionalFields, additionalFields)
+  }
+
+  // used by external ArtifactActivationStore SPI implementation
+  def activationToFileExtended(activation: WhiskActivation,
+                               context: UserContext,
+                               additionalFieldsForLogs: Map[String, JsValue] = Map.empty,
+                               additionalFieldsForActivation: Map[String, JsValue] = Map.empty) = {
+    val transcribedLogs = transcribeLogs(activation, additionalFieldsForLogs)
```

Comment:
Minor nit - As its a public method lets specify the return type explicitly. Which would be `Unit` in this case.

Reply:
Sure, done.

Reply:
ok, done